sim/rogue/talents.go

```diff
@@ -299,11 +298,10 @@ func (rogue *Rogue) applyCombatPotency() {
 }
```

Comment:
Combat Potency has ProcMask 20, so it should be `ProcMaskMeleeOH = ProcMaskMeleeOHAuto | ProcMaskMeleeOHSpecial`.